crates/typst/src/layout/layout.rs

```diff
@@ -43,8 +43,11 @@ use crate::syntax::Span;
 /// })
 /// ```
 ///
-/// Note that this function will provide an infinite width or height if one of
-/// the page width or height is `auto`, respectively.
+/// Note that the provided width or height will be infinite in case the
+/// corresponding page dimension is set to `{auto}`.
```

Comment:
I think this might be a slight bit better (not 100% sure):

```suggestion
/// Note that the width or height provided by `layout` will be infinite if the
/// corresponding page dimension is set to `{auto}`.
```

Reply:
Could also be worthwhile to add an example of this

Reply:
I agree with the rephrasing. Not sure this is worth adding an example, though. I fear it would increase the length of the - already long - documentation too much for nothing (and this sentence is clear enough in my opinion).